### ISSUE TITLE
Non-functional icons/spacers should be excluded from assistive technology

### ISSUE THREAD

**Issue by hauf-toni** [digitalfabrik/integreat-app#3636]
### Describe the Problem

When icons, SVGs  or images are purely decorative and don’t convey any meaningful information, screen readers may still try to interpret them or announce their filename or role. This creates unnecessary noise for users who rely on assistive technologies, making the content harder to understand and navigate.

### Describe Your Preferred Solution

Font icons, SVGs, or images used as spacers, decorations or when their purpose is already described by other content, should include a role="presentation" or role="none" attribute.

### Describe Possible Alternatives

/
### Related Issues

/

### Additional Information

Improves overall accessibiliy score.
Tested here: https://accessibe.com/

**Comment by lunars97** [comment]
we are already hiding decorative icons and images by using empty `alt`

**Comment by steffenkleinle** [comment]
I just did an audit with the same tool and I am a little confused:
<img width="2520" height="1220" alt="Image" src="https://github.com/user-attachments/assets/e02fab3b-0a79-425f-8af9-afcc9042d167" />

As you can see from the code snapshots of failed elements we already use the `<main>` tag correctly, so I think this is a false positive. I'd therefore reject the issue if there are no objections.

**Comment by steffenkleinle** [comment]
I just ran an audit with the same tool and for me this passes for integreat.app:

<img width="2490" height="120" alt="Image" src="https://github.com/user-attachments/assets/daf6e0b5-9546-4714-a8ce-e47293975a47" />

@hauf-toni could you perhaps check again if this is still the case for you?

**Comment by hauf-toni** [comment]
I recommend we close this issue since it could not be reproduced @steffenkleinle
